File: cross-application-feature-development-management/Directories/HostingDirectory/HostingDirectory.cs
namespace cross_application_feature_development_management.Directories.HostingDirectory
{
    public class HostingDirectory
    {
        public string GetPath()
        {
            var hostingDirectoryPath = CommandLineArgs.GetByKey("--hosting-directory");
            return hostingDirectoryPath;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/HostingDirectory.fs
namespace cafdemwimu.console.Directories.Hosting

open cafdemwimu.console
open cafdemwimu.console.Helpers

[<AbstractClass; Sealed>]
type HostingDirectory private () =
    static member GetPath() =
        let hostingDirectoryPath = CommandLineArgs.GetByKey("--hosting-directory")
        hostingDirectoryPath

    static member GetWrappedPath(stringHelpers: StringHelpers) : string =
        let hostingDirectoryPath = HostingDirectory.GetPath()
        let wrappedHostingDirectory = stringHelpers.WrapInQuotationMarks(hostingDirectoryPath)
        wrappedHostingDirectory
